### DISCUSSION TITLE
@apply with Svelte & Sapper is not working inside components

### DISCUSSION THREAD

**Discussion by adoveil** [tailwindlabs/tailwindcss#2303] [Help]
Hello,

I am trying to use Tailwind with Svelte and Sapper.
I have installed `sveltejs/sapper-template#rollup`, updated some packages & installed Tailwind.

The `@tailwind...` conversion in css file is OK but I can not at all use the **`@apply`** directives inside components... No error in the console nor in the terminal. Just the _line-throughed_ directive in the developer tool.

Here is the playground : https://github.com/adoveil/test-sapper-tailwind-apply

Thank you very much.

![image](https://user-images.githubusercontent.com/1140697/92130278-d50e8580-ee04-11ea-8717-dd7c8850be35.png)

![image](https://user-images.githubusercontent.com/1140697/92130208-c1631f00-ee04-11ea-9738-713af1cc8496.png)

**Comment by babichjacob** [comment]
Tailwind's official Sapper setup example doesn't include [`svelte-preprocess`](https://github.com/sveltejs/svelte-preprocess), so you can't use `@apply` nor any Tailwind directives or other PostCSS features in `.svelte` files. You'll need to modify it to add that support or use a pre-made template from [the community Sapper templates list](https://github.com/sveltejs/integrations#sapper-templates-boilerplates). 

Transparency warning / disclaimer: I compiled the list and am the creator of 3 of the templates on it.
